xcube/core/dsio.py

```diff
@@ -458,9 +451,20 @@ def write(self,
               output_path: str,
               compress=True,
               cname=None, clevel=None, shuffle=None, blocksize=None,
-              chunksizes=None):
+              chunksizes=None,
+              client_kwargs=None,
+              **kwargs):
+        mode = "write"
+        root = None
+        path_or_store, root, client_kwargs = _get_path_or_store(output_path, client_kwargs, mode, root)
+
+        # keeping the code commented out below, because might be useful for makinging write able to change bucket policy
+        # if 'acl' in kwargs:
+        #     acl = kwargs.pop('acl')
```

Comment:
Remove comment, unless you need to keep it. If so, say why.

Reply:
Well, there is a comment describing, why it is kept just above :)